src/gui/properties/propertieswidget.h

```diff
@@ -81,6 +81,7 @@ public slots:
     void readSettings();
     void saveSettings();
     void reloadPreferences();
+    void displayFileListHeaderMenu();
```

Comment:
This function should be `const`.

Reply:
I [can't](https://github.com/qbittorrent/qBittorrent/blob/56eaddf7fc515cd8a07a077b724156b1986fa243/src/gui/properties/propertieswidget.cpp#L180).